modules/blake2b/test.js

```diff
@@ -1,116 +1,117 @@
-var test = require('tape')
-var blake2b = require('./')
-var vectors = require('./test-vectors.json')
+const test = require('tape');
+const blake2b = require('./');
+const vectors = require('./test-vectors.json');
 
-setup()
+setup();
 test('wait for ready', function (assert) {
   blake2b.ready(function () {
-    assert.end()
-  })
-})
-setup()
+    assert.end();
+  });
+});
+setup();
 
 function setup () {
   test('vectors', function (assert) {
-    vectors.forEach(function (v) {
-      var out = new Uint8Array(v.outlen)
-      var input = hexWrite(new Uint8Array(v.input.length / 2), v.input)
-      var key = v.key.length === 0 ? null : hexWrite(new Uint8Array(v.key.length / 2), v.key)
-      var salt = v.salt.length === 0 ? null : hexWrite(new Uint8Array(v.salt.length / 2), v.salt)
-      var personal = v.personal.length === 0 ? null : hexWrite(new Uint8Array(v.personal.length / 2), v.personal)
-
-      var expected = Buffer.from(hexWrite(new Uint8Array(v.out.length / 2), v.out))
-      var actual = Buffer.from(blake2b(out.length, key, salt, personal, true).update(input).digest(out))
-
-      assert.deepEquals(actual, expected)
-    })
-    assert.end()
-  })
+    for (let i = 0; i < vectors.length; i++) {
```

Comment:
Curious if there's a reason why we switched from `forEach` to a for loop?

Reply:
At 576 vectors, `for` becomes faster than `forEach` in most environments.

Reply:
interesting, I've never heard that. Do you have a source for that metric?

Reply:
Here's something I did for it awhile ago. https://dev.to/mmcshinsky/benchmarking-javascript-loops-and-methods-part-1-19cl

It would be faster on multiple runs if it were being pre-optimized but each testing runs once as well is its' own one time run environment.

Reply:
When you get into object looping, it gets even worse: https://dev.to/mmcshinsky/benchmarking-javascript-loops-and-methods-part-2-2pfi

Reply:
Interesting, thanks for clearing that up!